pallets/loans/rpc/src/lib.rs

```diff
@@ -0,0 +1,93 @@
+use std::sync::Arc;
+
+use codec::Codec;
+use jsonrpc_core::{Error as RpcError, ErrorCode, Result};
+use jsonrpc_derive::rpc;
+pub use pallet_loans_rpc_runtime_api::LoansApi as LoansRuntimeApi;
+use sp_api::ProvideRuntimeApi;
+use sp_blockchain::HeaderBackend;
+use sp_runtime::FixedU128;
+use sp_runtime::{generic::BlockId, traits::Block as BlockT};
+
+#[rpc]
+pub trait LoansApi<BlockHash, AccountId> {
+    #[rpc(name = "pallet_loans_get_account_liquidity")]
```

Comment:
`pallet` is not necessary https://github.com/open-web3-stack/open-runtime-module-library/blob/master/oracle/rpc/src/lib.rs#L15